File: backend-play/app/com/github/brane08/pagila/seedworks/entities/BaseModel.java
package com.github.brane08.pagila.seedworks.entities;

import javax.persistence.Column;
import javax.persistence.MappedSuperclass;
import java.io.Serializable;
import java.time.Instant;

@MappedSuperclass
public class BaseModel implements Serializable {
	@Column(name = "last_update", updatable = false, insertable = false)
	public Instant lastUpdate;
}

File: backend-micronaut/src/main/kotlin/com/github/brane08/pagila/models/BaseModel.kt
package com.github.brane08.pagila.models

import java.time.Instant
import javax.persistence.Column
import javax.persistence.MappedSuperclass

@MappedSuperclass
open class BaseModel {
	@Column(name = "last_update", updatable = false, insertable = false)
	var lastUpdate: Instant? = null
}